test/utils/LogCapturing.scala

```diff
@@ -21,7 +21,7 @@ import ch.qos.logback.classic.{Level, Logger => LogbackLogger}
 import ch.qos.logback.core.read.ListAppender
 import play.api.LoggerLike
 
-import scala.collection.JavaConverters._
+import scala.jdk.CollectionConverters._
 
```

Comment:
This file should be deleted and its usages replaced with:

uk.gov.hmrc.play.bootstrap.tools.LogCapturing